spec/support/shared_contexts/authorization.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.shared_context "acting as signed-in user" do
+  let(:acting_user) { FactoryBot.create(:user) }
+end 
+
+RSpec.shared_context "acting as sysadmin" do
+  let(:acting_user) { FactoryBot.create(:user, :sys_admin) }
+end
+
+RSpec.shared_context "acting as admin" do
+  let(:acting_user) { FactoryBot.create(:user, :admin) }
+end
```

Comment:
Have to admit, i've grown to dislike globally shared contexts, particularly those that define `lets` referenced in specs. I find the cognitive load involved in jumping back and forth between spec, local context, shared context and sometimes other connected contexts to be quite heavy.

I really like the local ones in `user|person_policy_spec` because the context names provide more meaning that the lets therein. These ones definitely also provide meaning, but i'm finding it difficult have certainty just from the names exactly what they do. Perhaps this is case of needing to get used to the terminology ('acting-as', 'signed-in', etc) and once i do, my mind won't want to jump around. OTOH, i wonder if inlining these ends up being easier to grok, even at the cost of some more duplication?

The ones below seem widely used in request specs so probably worth their weight.